lib/src/widgets/chat/ds_unsupported_content_message_bubble.widget.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:blip_ds/blip_ds.dart';
+import 'package:flutter/material.dart';
+
+class DSUnsupportedContentMessageBubble extends StatelessWidget {
+  final DSAlign align;
+  final Widget? leftWidget;
+  final String? text;
+
+  const DSUnsupportedContentMessageBubble(
+      {Key? key, required this.align, this.leftWidget, this.text})
+      : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return DSMessageBubble(
+      align: align,
+      child: Row(
+        mainAxisSize: MainAxisSize.min,
+        children: [
+          leftWidget ??
+              Icon(
+                Icons.block,
```

Comment:
This icon cannot be from material. We need to use design system icons.

Reply:
done!